pkg/agent/billing/billing.go

```diff
@@ -146,31 +155,44 @@ func (s *metricsState) collect(conf *Config, store VMStoreForNode, metrics PromM
 	vmsOnThisNode := store.ListIndexed(func(i *VMNodeIndex) []*vmapi.VirtualMachine {
 		return i.List()
 	})
+	logger.Info(fmt.Sprintf("num_vms: %d", len(vmsOnThisNode)))
 	for _, vm := range vmsOnThisNode {
 		endpointID, isEndpoint := vm.Labels[EndpointLabel]
 		metricsBatch.inc(isEndpointFlag(isEndpoint), autoscalingEnabledFlag(api.HasAutoscalingEnabled(vm)), vm.Status.Phase)
-		if !isEndpoint {
-			// we're only reporting metrics for VMs with endpoint IDs, and this VM doesn't have one
-			continue
-		}
+		// if !isEndpoint {
+		// 	// we're only reporting metrics for VMs with endpoint IDs, and this VM doesn't have one
+		// 	continue
+		// }
 
 		if !vm.Status.Phase.IsAlive() || vm.Status.CPUs == nil {
+			logger.Info("Skipping gathering metrics for VM")
```

Comment:
this log line doesn't have any information about the VM. tbh I think this is _probably_ not worth adding, even if it had more info?

Reply:
Removed